backend/db/seeders/20240521075706-demo-Review.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+/** @type {import('sequelize-cli').Migration} */
+const {Review} = require('../models');
+//define schema
+let options = {}
```

Comment:
you might need to add:

```js
options.tableName = "Reviews";
```

Reply:
![Screenshot 2024-05-23 004837](https://github.com/360devin360/Sk8nBnB/assets/110138860/95b9be9b-7434-4c8c-80a7-14429da6d10a)